src/main.ts

```diff
@@ -171,6 +197,24 @@ export async function activate(context: vscode.ExtensionContext): Promise<CSharp
     };
 }
 
+function isSupportedPlatform(platform: PlatformInformation): boolean {
+    if (platform.isWindows()) {
+        return platform.architecture === "x86" || platform.architecture === "x86_64";
```

Comment:
Do you know if we get accurate information on **Windows** ARM64? Technically the language service side of the extension should work there. But if `platform.architecture` lies and says we are on x86 Windows then it doesn't matter.

Reply:
Just added a check for ARM64 on windows. It would always default to x86_64 if it is not just x86.

Reply:
Do things work with the language service on ARM64 Windows? If so, maybe we shouldn't block it here, but rather just fail debugger launch?

Reply:
I don't see a win-arm64 package in omnisharp-roslyn.

@JoeRobich Does the language service run on Windows ARM64?

Reply:
There presently is no WinArm64 package. I've been looking into LinuxArm64 but windows has not been on my radar. =)

Reply:
Yes, but on Windows arm64, the x86 OmniSharp will work.